actionpack/lib/action_dispatch/constants.rb

```diff
@@ -11,12 +11,14 @@ module Constants
       LOCATION = "Location"
       FEATURE_POLICY = "Feature-Policy"
       X_REQUEST_ID = "X-Request-Id"
+      SERVER_TIMING_HEADER = "Server-Timing"
```

Comment:
We should probably make these just `SERVER_TIMING` since we're moving them anyways